public/app/features/alerting/routes.tsx

```diff
@@ -235,6 +235,17 @@ const unifiedRoutes: RouteDescriptor[] = [
       () => import(/* webpackChunkName: "AlertingRuleForm"*/ 'app/features/alerting/unified/RuleEditor')
     ),
   },
+  {
```

Comment:
It seems to me the current number of routes is a bit confusing. We have
- `/alerting/new`
- `/alerting/new/grafana`
- `/alerting/new/cloud-alerting`
- `/alerting/new/cloud-recording`
What if we have `/alerting/new/alerting` and `/alerting/new/recording`?

Reply:
And what about Loki/Mimir? (currently `cloud-alerting`), shouldn't we add a new route for it as well?

Reply:
I think both Grafana and Loki/Mimir should be under `/alerting/new/alerting`. 
My rationale is that you click `Create Alert Rule` or `More -> Create recording rule` so we have two different entry points. Additionally, if we remove the `Recording rule` from the rule type selection, this routing change makes more sense. Yet it makes less sense if we do not remove the `Recording rule` from the type selection.

Reply:
It sounds good to me, removed the button and changed the routes as you suggested 👍